shell_integration/nautilus/setappname.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 
 # this script replaces the line
-#  appname = 'Nextcloud'
+#  appname = 'ownCloud'
 # with the correct branding name in the syncstate.py script
 # It also replaces the occurrences in the class name so several
 # branding can be loaded (see #6524)
-sed -i.org -e "s/Nextcloud/$1/g" syncstate.py
+sed -i.org -e "s/ownCloud/$1/g" syncstate.py
```

Comment:
please do the opposite and make sure that every strings that would need to be replaced by the script `setappname.sh` is `Nextcloud`
doing the opposite is not fixing the bug at the proper level

Reply:
Hello @mgallien,

thanks for your feedback. Of course I can change every string to "Nextcloud". I also thought of this first. But I think the `setappname.sh` script was exactly written for this purpose.

I don't know if you did any modifications to the extension or if you just use the code from the ownCloud extension. If it's the second case, then you don't have to do any changes to the code. The `setappname.sh ` will handle the naming of all required variables.

So merging of new code from ownCloud will be much easier.

Please let me know your thoughts about this. Thank you.

Reply:
@mammo0 sorry for the big delay
we really want to remove all usages of ownCloud in part because that no longer makes sense but most importantly to remove confusion
can you replace all `ownCloud` strings by `Nextcloud` ?
that should make both extensions compatible and we would already have conflicts when cherry-picking from their repository

Reply:
see commit f9fd1832f94c8ee130901c5aabdf7c158e1a62e3